src/core/request.js

```diff
@@ -264,40 +273,11 @@ Request.prototype.callback = function(callback, successEvent, errorEvent) {
   return this;
 };
 
-Request.prototype.timeout = function(duration, callback) {
```

Comment:
Since this is a breaking change, can you add a not in the commit message that says it contains a breaking change?